docs/modules/ROOT/pages/how-tos/upgrade-v1-v2.adoc

```diff
@@ -0,0 +1,91 @@
+= Upgrade `component-cert-manager` from `v1.x` to `v2.x`
```

Comment:
The how-to page needs to be linked in the `partial/nav.adoc` file in order to pop up in the navigation

Reply:
done, thanks!